src/main/app/claims/models/claim.ts

```diff
@@ -162,4 +163,13 @@ export class Claim {
   private isSettlementReached (): boolean {
     return this.settlement && !!this.settlementReachedAt
   }
+
+  get pastDefendantPayImmediatelyDate (): boolean {
+    const currentDateSetTo4PM = MomentFactory.currentDate().hour(15)
+    return currentDateSetTo4PM > this.payImmediatelyDeadline()
+  }
+
+  private payImmediatelyDeadline (): Moment {
```

Comment:
Move that to response. It does not belong to claim. Also helps us getting errors when response is undefined